Makefile

```diff
@@ -207,3 +207,49 @@ check-torch-gpu: ## Check whether PyTorch works on GPU using VM with Custom Cont
 
 check-pipeline: ## Check whether the Beam pipeline can run on GPU using VM with Custom Container and DirectRunner
 	@./scripts/check-pipeline.sh
+
+create-flex-template: ## Create a Flex Template file using a Flex Template custom container
+	gcloud dataflow flex-template build $(TEMPLATE_FILE_GCS_PATH) \
+	--image $(CUSTOM_CONTAINER_IMAGE) \
+	--metadata-file ./flex/metadata.json \
+	--sdk-language "PYTHON" \
+	--staging-location $(STAGING_LOCATION) \
+	--temp-location $(TEMP_LOCATION) \
+	--project $(PROJECT_ID) \
+	--worker-region $(REGION) \
+	--worker-machine-type $(MACHINE_TYPE)
+
+run-df-gpu-flex: ## Run a Dataflow job using the Flex Template
```

Comment:
Should we have a non-gpu option?

Reply:
I have `--parameters device=GPU` as the option below. Users could change this target to run the job without GPUs if they want.